hack/cluster.sh

```diff
@@ -75,7 +76,12 @@ on_cluster_up() {
 	kubectl apply --force -f "$PROJECT_ROOT/hack/crds"
 
 	info "setup OLM"
-	operator-sdk olm install --verbose
+	[[ -f "$BIN_DIR/operator-sdk" ]] || {
+		err "operator-sdk not available"
+		info "run make tools"
+		return 1
+	}
```

Comment:
+1 Nice addition! 

but I don't think we should insist that operator-sdk is always installed to `"$BIN_DIR"` it is possible that user has installed elsewhere. We should 
1. validate if the binary exists and if the version is the right one. see makefile. Or just call `make operator-sdk` which already does this.
2. in main(), `export PATH="$BIN_DIR:$PATH"`

Reply:
> I don't think we should insist that operator-sdk is always installed to "$BIN_DIR" it is possible that user has installed elsewhere

we already recommend running `make tools` before performing any operations as it allows us to set up the required tools with the versions we want on the user's local machine. And it assumes installing those tools inside `BIN_DIR`. Why should we consider if operator-sdk is available or not and what version of operator-sdk is pre-installed on the user's machine?

Reply:
>  Why should we consider if operator-sdk is available or not and what version of operator-sdk is pre-installed on the user's machine?

If the operator-sdk is already installed, and matches the version, what is the in point of insisting it be installed in a specific path? This become even more of an issue if you use git work-trees.